contributing.md

```diff
@@ -1,3 +1,5 @@
+---
+---
```

Comment:
We should probably be using the metadata blocks to add things like page titles.

Reply:
The reason I shied away from this was I didn't want to add any additional workload for non-devs who are contributing to the Playbook. This is a very loosely-held opinion though

Reply:
It's an upgrade. If they don't do it, it will have the default title and all will be well :).

Reply:
OK, this is done now 👍